features/step_definitions/step_definetions.rb

```diff
@@ -0,0 +1,31 @@
+Given(/^the following user exist$/) do |table|
+  table.hashes.each do | user |
+    User.create(user)
+  end
+end
+
+Given(/^I am logged in as "([^"]*)"$/) do |name|
+  user = User.find_by(name: name)
+  login_as(user, scope: :user)
+  visit '/'
+end
+
+Given(/^I click on the "([^"]*)"$/) do |arg1|
+  click_link(arg1)
+end
+
+Given(/^I click on the "([^"]*)" button$/) do |button_name|
```

Comment:
You could refactor this step to work for both links and buttons

So if we have the following 2 steps

```gherkin
Given I click on the "login" link
Given I click on the "login" button
```

we could have a single step definition to match them

```ruby
Given(/^I click on the "([^"]*)" (?:link|button)$/) do |link_or_button_name|
  click_link_or_button link_or_button_name
end
```